lib/XTR/ImageProcessingTask.php

```diff
@@ -85,16 +86,23 @@ private function runImageProcessing()
                 $this->imageProcessingService->resizeImage($filePath, null, $resizeThreshold);
             }
 
+            // Clears file status cache
+            clearstatcache(true, $filePath);
+
             // Release image and save
-            // Work out the MD5
-            $media->md5 = md5_file($libraryLocation . $media->storedAs);
+            // Work out the MD5 and Filesize
+            $media->md5 = md5_file($filePath);
```

Comment:
Maybe it would make more sense to set these inside the release() method?

so we call: ```$media->release(md5_file($filePath), filesize($filePath));```

and then inside release, set the properties (including released = 1)